tests/myapp/models.py

```diff
@@ -4,7 +4,44 @@
 from publisher.models import PublisherModel
 
 
+try:
+    import parler
+except ImportError:
+    PARLER_INSTALLED=False
+else:
+    PARLER_INSTALLED = True
+
+
+try:
+    import aldryn_translation_tools
+except ImportError as err:
+    TRANSLATION_TOOLS_INSTALLED=False
```

Comment:
same here, use `aldryn_translation_tools = None` for the except part. Also, remove the `as err`.

Reply:
done in https://github.com/andersinno/django-model-publisher-ai/pull/14/commits/0379bcb3525089541fd33d8bb9510e0bf4f734e5